packages/compass-e2e-tests/tests/instance-databases-tab.test.ts

```diff
@@ -38,12 +42,15 @@ describe('Instance databases tab', function () {
     }
   });
 
-  it('links database cards to the database collections tab', async function () {
+  it.skip('links database cards to the database collections tab', async function () {
```

Comment:
This fails almost every time in CI. Here is the screen inside clickVisible at the moment before it actually clicks the card:

![database-card](https://user-images.githubusercontent.com/69737/155115019-9183c9fc-dac2-438f-8385-6fd1918bd690.png)
 
I'm skipping it for now until we can figure out what is going on.

Reply:
https://jira.mongodb.org/browse/COMPASS-5529